samples/hello-world/package.json

```diff
@@ -10,6 +10,7 @@
     "start:ios": "npm run rn-cli -- run-ios",
     "start:web": "cross-env platform=web webpack-dev-server --config=web/webpack/dev.js --progress --colors --mode=development",
     "start:rn-dev-server": "npm run rn-cli -- start --reset-cache",
+    "build": "tsc",
```

Comment:
Is there a case where this command is needed?

Reply:
I couldn't find a way to compile to JS. 
I suspect the run-<platform> may do it, but I can't run "run-windows" (it fails miserably).

Reply:
are you talking about `npm run start:windows`?

Reply:
Yes. The msbuild part of it fails due to the pfx certificate being encrypted with a password, I suspect a password of yours since the file comes from you.
I admit a lot of this msbuild/VS is (non fulfilling) magic to me, so I don't know how to fix. 
Running from within VS works fine.

Reply:
You need to generate your [self-signed certificate](https://docs.microsoft.com/en-us/powershell/module/pkiclient/new-selfsignedcertificate?view=win10-ps). Basically [metro-react-native-babel-preset](https://github.com/facebook/metro/blob/master/packages/metro-react-native-babel-preset/src/configs/main.js#L16) understands TypeScript due to [@babel/plugin-transform-typescript](https://github.com/facebook/metro/blob/master/packages/metro-react-native-babel-preset/src/configs/main.js#L173), and some of example use this feature.

Reply:
It's not covered in the readme's, hence my confusion (assuming I'm doing something wrong).
I'll see what I can do on Monday, today I'm past the quota allocated for this. Thanks.

Reply:
Also, there is command to run dev server - `npm run start:rn-dev-server`

Reply:
True, I used that.

Reply:
Tried with local self signed certs and all three projects that use "npm run start:windows" (hello-world, hello-world-js, ImageList) work correctly from command line in Windows now.